open-backport-pull-request/src/run.ts

```diff
@@ -55,34 +72,57 @@ const hasDiff = async (params: hasDiffParams): Promise<boolean> => {
 
 type openPullRequestParams = {
   headBranch: string
+  headSha: string
   baseBranch: string
   context: Context
   octokit: Octokit
 }
+
 const openPullRequest = async (params: openPullRequestParams): Promise<string | undefined> => {
+  // Create a working branch so that we can edit it if conflicted.
+  // Generally, the head branch is protected and cannot be edited.
+  const workingBranch = `backport-${params.headBranch.replaceAll('/', '-')}-${Date.now()}`
+  await params.octokit.rest.git.createRef({
+    owner: params.context.repo.owner,
+    repo: params.context.repo.repo,
+    ref: `refs/heads/${workingBranch}`,
+    sha: params.headSha,
+  })
+
+  core.info(`Creating a pull request ${workingBranch} -> ${params.baseBranch}`)
+  const pr = await params.octokit.rest.pulls.create({
+    owner: params.context.repo.owner,
+    repo: params.context.repo.repo,
+    head: workingBranch,
+    base: params.baseBranch,
+    title: `Backport ${params.headBranch} into ${params.baseBranch}`,
+    body: `Created from https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}`,
+  })
```

Comment:
We don't need to check `UnprocessableEntityError` here, because the working branch name is unique.